src/unit_tests/kubernetes_api_objects/config_map.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright 2022 VMware, Inc.
+// SPDX-License-Identifier: MIT
+use crate::kubernetes_api_objects::config_map::*;
+use crate::kubernetes_api_objects::object_meta::*;
+use crate::kubernetes_api_objects::resource::*;
+use crate::pervasive_ext::string_map::*;
+use vstd::prelude::*;
+use vstd::string::*;
+
+verus! {
+
+#[test]
+#[verifier(external)]
+pub fn test_set_metadata() {
+    println!("Testing set_metadata()...");
+    let object_meta_gen = || {
```

Comment:
I don't think you need this closure. Just use object_meta.clone() -- if it does not implement clone then you can implement one for it. See https://github.com/vmware-research/verifiable-controllers/blob/main/src/kubernetes_api_objects/config_map.rs#L43